QgisModelBaker/gui/workflow_wizard/project_creation_page.py

```diff
@@ -712,6 +716,50 @@ def _inheritance(self):
             if setting_record["tag"] == "ch.ehi.ili2db.inheritanceTrafo":
                 return setting_record["setting"]
 
+    def _multigeom_gpkg(self):
+        # this concerns only geopackage
+        if not (self.workflow_wizard.import_schema_configuration.tool & DbIliMode.gpkg):
+            return False
+
+        # and when this geopackage has multiple geometry columns in a table
+        if len(self.db_connector.multiple_geometry_tables()) == 0:
+            return False
+
+        if int(gdal.VersionInfo("VERSION_NUM")) < 3080000:
+            self.gpkg_multigeometry_label.setText(
+                """
+                <html><head/><body style="background-color:powderblue;">
+                     <p><b>This GeoPackage contains at least one table with multiple geometries</b></p>
+                    <p>These tables require <span style=" font-weight:600;">GDAL version &gt;= 3.8</span> to run in QGIS.<br/>Your current QGIS version is <span style=" font-weight:600;">{qgis_version}</span> with GDAL <span style=" font-weight:600;">{gdal_version}</span>.</p>
+                    <p>Means this won't work.</p>
+                </body></html>
+                """.format(
+                    qgis_version=Qgis.QGIS_VERSION,
+                    gdal_version=gdal.VersionInfo("RELEASE_NAME"),
+                )
+            )
+            self.create_project_button.setDisabled(True)
+        else:
+            self.gpkg_multigeometry_label.setText(
+                """
+                <html><head/><body style="background-color:powderblue;">
+                    <p><b>This GeoPackage contains at least one table with multiple geometries</b></p>
+                    <p>These tables require <span style=" font-weight:600;">GDAL version &gt;= 3.8</span> to run in QGIS.<br/>Your current QGIS version is <span style=" font-weight:600;">{qgis_version}</span> with GDAL <span style=" font-weight:600;">{gdal_version}</span>.</p>
+                    <p>But note that others with lower 3.8 version <span style=" font-weight:600;">will not be able </span>to read such tables in the created QGIS project.</p>
+                </body></html>
+                """.format(
+                    qgis_version=Qgis.QGIS_VERSION,
+                    gdal_version=gdal.VersionInfo("RELEASE_NAME"),
+                )
+            )
+        return True
+
+    def _multiple_geometry_gpkg_table(self):
+        if self.db_connector.multiple_geometry_tables() > 0:
+            return True
+
+        return false
```

Comment:
`NameError: name 'false' is not defined` :)

Reply:
yes, same leftover like above